main.cpp

```diff
@@ -16,21 +16,21 @@ extern mat<4,4> Projection;
 
 struct Shader : IShader {
     const Model &model;
-    vec3 l;               // light direction in normalized device coordinates
+    vec3 l;               // light direction in view coordinates
```

Comment:
I am curious, was there a reason why the lighting was originally done in NDC?  From what I understand, it can also be done in World or View coordinates, as long as we stay consistent, right?

Sorry, I am not really commenting on the change, I am trying to understand the motivation for choosing one frame or reference over another.